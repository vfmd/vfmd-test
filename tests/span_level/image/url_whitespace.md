 * ![link](url 1)
 * ![link](url
   1)
 * ![link](<url 1>)
 * ![link](<url 
   1>)
 * ![link][ref1]
 * ![link][ref2]
 * ![link][ref3]
 * ![link][ref4]

[ref1]: url 1
[ref2]: <url 1>

[ref3]: url
1

[ref4]: <url/
1>
