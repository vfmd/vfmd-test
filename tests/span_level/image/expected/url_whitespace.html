
<ul>
  <li>![link](url 1)</li>
  <li>![link](url
1)</li>
  <li><img src="url1" alt="link" /></li>
  <li><img src="url1" alt="link" /></li>
  <li><img src="url" alt="link" /></li>
  <li><img src="url1" alt="link" /></li>
  <li><img src="url" alt="link" /></li>
  <li>![link][ref4]</li>
</ul>
<p>1</p>

<p>[ref4]: &lt;url/
1&gt;</p>

